openwisp_users/base/models.py

```diff
@@ -215,6 +215,15 @@ class BaseOrganizationUser(models.Model):
     class Meta:
         abstract = True
 
+    def clean(self):
+        if getattr(self, 'organizationowner', None) and not self.is_admin:
```

Comment:
why not use the helpers method we created? We should avoid queries as much as possible:

```python
if self.user.is_owner(self.organization) and not self.is_admin:
    ...
```

or alternatively:

```python
if str(self.organization.pk) in self.user.organizations_owned and not self.is_admin:
    ...
```

See my comment below about `assertNumQueries`, can you quickly expriment with the 3 approaches and let me know which one generates less queries?

Reply:
@nemesisdesign all 3 cases use just one query. I think `getattr(self, 'organizationowner', None)` didn't generate extra queries because it was gotten using `select_related`. But we can't and shouldn't depend on that so I am using `self.user.is_owner(self.organization)`